README.rst

```diff
@@ -79,6 +79,27 @@ Alternatively, glog can be incorporated into using the CMake command
 project by replacing the :cmake:`find_package` call from the previous snippet by
 :cmake:`add_subdirectory`. The :cmake:`glog::glog` target is in this case an
 :cmake:`ALIAS` library target for the ``glog`` library target.
+Here is an example which using :cmake:`FetchContent` and :cmake:`add_subdirectory`
```

Comment:
Do we really need to show a complete example for how to use `FetchContent`? This is essentially an exact mirror of the example from [CMake reference documentation](https://cmake.org/cmake/help/latest/module/FetchContent.html#overview). In that sense, a single sentence with a link to `FetchContent` example should be sufficient.